### PR TITLE
cig 0.1.3 (new formula)

### PR DESCRIPTION
**Pull request by stevenjack**
* Adds formula for [cig](https://www.github.com/stevenjack/cig)

### CODE REVIEW

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: Library/Formula/cig.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  depends_on "go" => :build
+
+  def install
+    ENV["AUTO_GOPATH"] = "1"
+    system "go", "build", "-o", "cig"
+    bin.install "cig"
+  end
+
+  test do
+    system "#{bin}/cig", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/cig.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/cig.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Cig < Formula
+  homepage "https://github.com/stevenjack/cig"
+  url "https://github.com/stevenjack/cig/archive/v0.1.1.tar.gz"
+  sha1 "4ce0822d4987e39f4a9c2d5da7234293d1404557"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["AUTO_GOPATH"] = "1"
+    system "go", "build", "-o", "cig"
```
Comment (mikemcquaid) [inline Library/Formula/cig.rb]: Does this have any dependencies?

**Comment by mikemcquaid** [comment]
Install is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26861/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_cig/

also, please fix the `audit` warnings: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26861/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_cig___strict/

**Comment by DomT4** [comment]
Ping.

**Comment by stevenjack** [comment]
@mikemcquaid Sorry I've been away on holiday away from my laptop, I've not finished the PR yet but will look to get it done by the end of the week. In regards to your questions:

1. The deps are stored in the project using `godeps` so in that sense there aren't any external dependencies.
2. I'll modify the test after having a look at the examples you mentioned.

Thanks for the feedback

**Comment by stevenjack** [comment]
@mikemcquaid @DomT4 this should be good to go now, all tests passing and have updated the test to actually exercise the application.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/e6233a382a41f8fb365bd05f0af51f1b040d2b60. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**